src/wmtk/operations/tet_mesh/CMakeLists.txt

```diff
@@ -2,8 +2,10 @@ set(SRC_FILES
     EdgeOperationData.hpp
     EdgeOperationData.cpp
 
-    CollapseNewAttributeStrategy.hpp
-    CollapseNewAttributeStrategy.cpp
+    # CollapseNewAttributeTopoInfo.hpp
```

Comment:
@JcDai is about to / in the process of affecting these files. @teseoch could you make sure to discuss this change with @JcDai ?